pkg/artifactory/resource/webhook/resource_artifactory_webhook.go

```diff
@@ -161,15 +161,17 @@ func ResourceArtifactoryWebhook(webhookType string) *schema.Resource {
 				handlers := v.(*schema.Set).List()
 				for _, handler := range handlers {
 					h := handler.(map[string]interface{})
-
-					webhookHandler := Handler{
-						HandlerType:       "webhook",
-						Url:               h["url"].(string),
-						Secret:            h["secret"].(string),
-						Proxy:             h["proxy"].(string),
-						CustomHttpHeaders: unpackCustomHttpHeaders(h["custom_http_headers"].(map[string]interface{})),
+					// use this to filter out weirdness with terraform adding an extra blank webhook in a set
+					if h["url"].(string) != "" {
```

Comment:
In other languages, I prefer to use `continue` as a short circuit. i.e.

```go
if h["url"].(string) == "" {
    continue
}
```

Just my preference, not a blocker.